eventually/vote/models.py

```diff
@@ -56,16 +75,18 @@ def to_dict(self):
         :Example:
          {
              'id': 13,
+             'event':1,
              'is_active': True,
              'is_extended': True,
              'title': 'Title of vote',
-             'type': 'M',
+             'type': 0,
              'created_at': 1509540116,
              'updated_at': 1509540116,
          }
         """
         return {
-            'id':self.id,
+            'id': self.id,
+            'event': self.event.id,
```

Comment:
you have event null=True so it can be empty.
use  self.event.id if self.event else None,